tests/phpunit/integration/Core/Consent_Mode/REST_Consent_Mode_ControllerTest.php

```diff
@@ -67,6 +69,8 @@ class REST_Consent_Mode_ControllerTest extends TestCase {
 
 	public function set_up() {
 		parent::set_up();
+		// Avoid unexpected results when running locally.
+		$this->mock_installed_plugins();
```

Comment:
This fixes an issue I often saw when running tests locally with the `wp-consent-api` plugin installed.